test/DynamoCoreTests/LibraryTests.cs

```diff
@@ -232,6 +233,8 @@ public void MethodWithRefOutParams_NoLoad()
         [Category("UnitTests")]
         public void DumpLibraryToXmlZeroTouchTest()
         {
+            var searchViewModel = new SearchViewModel(null, new NodeSearchModel());
+
```

Comment:
Nice, local copy of `SearchViewModel`. I think if you move this test out so it doesn't require construction of the main `DynamoViewModel`, then this test will be free of having to create `DynamoViewModel` without real needs.